mindjunkies/forums/migrations/0001_initial.py

```
@@ -10,243 +10,97 @@
-                    ),
-                ),
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('comment', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='forums.forumcomment')),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
             ],
         ),
+        migrations.AddField(
+            model_name='forumcomment',
+            name='likes',
+            field=models.ManyToManyField(related_name='likedComments', through='forums.LikedComment', to=settings.AUTH_USER_MODEL),
```

Ruff (0.8.2)

Line too long (131 > 119)

(E501)